.github/workflows/size_check.yml

```diff
@@ -20,18 +27,24 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: main_ref
+
+      - name: Install dependencies
+        run: |
+          pip3 install Pillow
  
-      - run: |
-          echo ${{ secrets.TFLM_BOT_PACKAGE_READ_TOKEN }} | docker login ghcr.io -u tflm-bot --password-stdin
-          docker run --rm -v `pwd`:/opt/tflite-micro ghcr.io/tflm-bot/xtensa:latest /opt/tflite-micro/ci/test_size.sh
-          
-      - name: compare bss value and commit
+      - name: Build binary
+        run: |
+          tensorflow/lite/micro/tools/ci_build/test_size.sh
+
+      - name: Update size history and commit
         run: |
-          ci/size_comp.py -c main_ref/.ci_vars/fusion_f1.json ci/log.txt
-          rm -rf .ci_vars/fusion_f1.json
-          ci/size_comp.py -t ci/log.txt .ci_vars/fusion_f1.json
+          # Disable automatic failure from size comparison for now:
+          #  we do not want all size change to fail automatically.
+          # ci/size_comp.py -c main_ref/tensorflow/lite/micro/tools/ci_build/binary_size_history/binary_size.json ci/size_log.txt
```

Comment:
Dead code?

Reply:
Done. We might want to bring it back in future. But yes, not used for now.